packages/stack.rb

```diff
@@ -48,7 +34,7 @@ def self.postinstall
     puts "source ~/.bashrc".lightblue
     puts
     puts "To completely uninstall stack, execute the following:".lightblue
-    puts "crew remove cabal ghc stack".lightblue
+    puts "crew remove stack".lightblue
     puts "rm -rf #{CREW_PREFIX}/share/stack".lightblue
```

Comment:
This directory was removed for me when I ran `crew remove stack` so maybe we can get rid of this line.

Reply:
I just included it as part of the entire process.  Probably doesn't hurt to leave it for clarity.

Reply:
Yeah, this directory can be populated with other stack tools like ghc, for one.  Definitely should leave it.